service/src/main/resources/db/migration/R__message_account_name_view.sql

```diff
@@ -1,14 +1,16 @@
 DROP VIEW IF EXISTS message_account_name_view;
 CREATE VIEW message_account_name_view(
     id,
+    type,
     account_name
 ) AS (
 SELECT
     acc.id,
+    acc.type,
     COALESCE(
         nullif(concat_ws(' - ', d.name, dg.name), ''),
         nullif(concat_ws(' ', p.last_name, p.first_name), ''),
-        concat_ws(' ', e.last_name, coalesce(e.preferred_first_name, e.first_name))
```

Comment:
Instead of making changes to `message_account_name_view`, you should probably make the changes to `message_account_view` and use it instead. In my experience, it's quite a bit faster and should be used everywhere. I just didn't have time to change all places to use it back then.